test_distance_consistency fails on 32bit

```
[  551s] ____________________ DotIndexTest.test_distance_consistency ____________________
[  551s] 
[  551s] self = <dot_index_test.DotIndexTest testMethod=test_distance_consistency>
[  551s] 
[  551s]     def test_distance_consistency(self):
[  551s]         n, f = 1000, 3
[  551s]         i = AnnoyIndex(f, 'dot')
[  551s]         for j in range(n):
[  551s]             i.add_item(j, numpy.random.normal(size=f))
[  551s]         i.build(10)
[  551s]         for a in random.sample(range(n), 100):
[  551s]             indices, dists = i.get_nns_by_item(a, 100, include_distances=True)
[  551s]             for b, dist in zip(indices, dists):
[  551s]                 self.assertAlmostEqual(dist, numpy.dot(
[  551s]                     i.get_item_vector(a),
[  551s]                     i.get_item_vector(b)
[  551s]                 ))
[  551s] >               self.assertEqual(dist, i.get_distance(a, b))
[  551s] E               AssertionError: 5.229551315307617 != 5.229551142363135
[  551s] 
[  551s] dot_index_test.py:158: AssertionError

```

--- a/test/dot_index_test.py
+++ b/test/dot_index_test.py
@@ -155,4 +155,4 @@ class DotIndexTest(TestCase):
                     i.get_item_vector(a),
                     i.get_item_vector(b)
                 ))
-                self.assertEqual(dist, i.get_distance(a, b))
+                self.assertAlmostEqual(dist, i.get_distance(a, b))

